spec/features/users_creates_new_account_spec.rb

```diff
@@ -17,42 +40,72 @@
       ))
   end
 
-  scenario "views show page" do
-    user = create(:user)
+  scenario "successfully as admin" do
```

Comment:
What does `scenario "successfully as admin" do` this scenario do?

Reply:
Changed to admin creates new admin account. Done in sha: b2bcac4